src/Pages/Calculator/EndScreen.jsx

```diff
@@ -29,6 +33,7 @@ const EndScreen = props => {
             <Grid item className={classes.bodyGrid}>
                 {props.body &&
                     props.body.map((b, idx) => {
+                        console.log(props);
```

Comment:
Let's remove any stray `console.logs` before merging into staging branch : ) This was the MVP for the Hackathon, so it's great to know that we have all of the data at this point in the user's journey through the calc.